ffmpeg_normalize/_ffmpeg_normalize.py

```diff
@@ -1,45 +1,45 @@
-import os
 import json
-from numbers import Number
-from typing import List, Literal, Union
+import os
+from typing import TYPE_CHECKING, List, Literal, Union
+
 from tqdm import tqdm
 
-from ._cmd_utils import get_ffmpeg_exe, ffmpeg_has_loudnorm
-from ._media_file import MediaFile
+from ._cmd_utils import ffmpeg_has_loudnorm, get_ffmpeg_exe
 from ._errors import FFmpegNormalizeError
 from ._logger import setup_custom_logger
+from ._media_file import MediaFile
+
+if TYPE_CHECKING:
+    from ._streams import LoudnessStatisticsWithMetadata
 
 logger = setup_custom_logger("ffmpeg_normalize")
 
-NORMALIZATION_TYPES = ["ebu", "rms", "peak"]
-PCM_INCOMPATIBLE_FORMATS = ["mp4", "mp3", "ogg", "webm"]
-PCM_INCOMPATIBLE_EXTS = ["mp4", "m4a", "mp3", "ogg", "webm", "flac", "opus"]
```

Comment:
I am not sure if I would like to merge this particular part.

Formats are different from extensions. The user can specify formats in terms of ffmpeg's `-f` option, whereas if not specified, ffmpeg automatically determines the format based on the extension.

At the moment `PCM_INCOMPATIBLE_FORMATS` is a subset of `PCM_INCOMPATIBLE_EXTS`, but not all extensions might be valid format names for ffmpeg.

What do you think?

Reply:
I think you have a point. `m4a` is an ext but not a format so the code should document this fact. My next commit addresses your changes. I also added `oga` to the ext and format set since I noticed it also does not support pcm